ts/features/bonus/cgn/components/merchants/discount/CgnDiscountHeader.tsx

```diff
@@ -36,21 +31,28 @@ export const CgnDiscountHeader = ({
     setAccessibilityFocus(ref, 400 as Millisecond);
   }, [ref]);
 
+  const { header: headerStyle } = useCgnStyle();
+
+  const backgroundColor = isNew
+    ? headerStyle.new.backgroundColor
+    : headerStyle.default.backgroundColor;
+
+  const foreground: IOColors = isNew
+    ? headerStyle.new.foreground
+    : headerStyle.default.foreground;
```

Comment:
Since we are extracting `backgroundColor` and `foreground` values, what do you think to use this approach?
 ```suggestion
  const { backgroundColor, foreground } = isNew
    ? headerStyle.new
    : headerStyle.default;
```